modules/sso/install

```diff
@@ -10,7 +10,16 @@ pushd $JBOSS_HOME/bin
 ./jboss-cli.sh --file=keycloak-install.cli
 popd
 
+# Clean up the left-over content of the history directory if it exists
+XML_HISTORY_CURRENT_DIR="$JBOSS_HOME/standalone/configuration/standalone_xml_history/current"
+if [ -d "$XML_HISTORY_CURRENT_DIR" ]; then
+  rm -rf "$XML_HISTORY_CURRENT_DIR"
+  if [ "$?" -ne "0" ]; then
```

Comment:
Everything added below is not needed because we run the script with `-e` which terminates it if any of commands fail.

Reply:
Nice catch! Will update yet.